### PR TITLE
Closes #4336 Make Media Notification Icons White As Per Android Doc R…

### LINKED ISSUE #4336: Media notification icon color does not adapt on Android 5
Fenix issue: https://github.com/mozilla-mobile/fenix/issues/5154

### PR DESCRIPTION
**Pull request by rocketsroger**
…ecommendation

---

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- [ ] **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

**Comment by rocketsroger** [comment]
Tested on Android 10 and Pie with dark and light themes.  I could not reproduce the issue on Android 5.0 and 5.1 emulator.  Nexus 5 was also not showing the same issue.  I'll be asking for testing help when this lands in Fenix to confirm the fix.  But based on the Android Doc this is the correct fix.

File: components/feature/media/src/main/res/drawable/mozac_feature_media_action_pause.xml [hunk 1]
```diff
@@ -4,6 +4,6 @@
     android:viewportWidth="24"
     android:viewportHeight="24">
   <path
-      android:fillColor="#FF000000"
+      android:fillColor="#FFFFFFFF"
```
Comment (NotWoods) [inline components/feature/media/src/main/res/drawable/mozac_feature_media_action_pause.xml]: nit: I think omitting the alpha works too
```suggestion
      android:fillColor="#FFFFFF"
```

Comment (rocketsroger) [reply components/feature/media/src/main/res/drawable/mozac_feature_media_action_pause.xml]: Agreed.  I'll change it to omit alpha.  Thanks,

**Pull request merged**